src/stories/tag/Tag.tsx

```diff
@@ -39,7 +49,23 @@ export const Tag: FC<TagProps> = ({
             size === 'compact' && typography.labelXXSmall,
             className,
         )}
+        data-status={colorLevel}
     >
-        {children}
+        {size !== 'small' && (
+            <div>
+                {children}
+            </div>
+        )}
+        {size === 'small' && (
+            <div className={styles.icon}>
+                <RemoveFromDOM when={!!icon}>
+                    <Icon icon={Check} size={12} />
```

Comment:
again, this kind of thing is best done by setting a default value for the `icon` prop instead of adding all of this extra code

Reply:
Yeah originally I had set this up to have different icons for different states, but then moved away from that and forgot to move the code back to simple default, made that change